PT/asciidoc/src/01_introduction_and_goals.adoc

```diff
@@ -0,0 +1,100 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-introduction-and-goals]]
+== Introdução e Objetivos
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+Descreve os requisitos relevantes e as forças motrizes que os arquitetos de software e a equipe de desenvolvimento devem considerar.
+Isso inclui
+
+* objetivos de negócios subjacentes,
+* recursos essenciais,
+* requisitos funcionais essenciais,
+* objetivos de qualidade para a arquitetura e
+* partes interessadas relevantes e suas expectativas
+****
+endif::arc42help[]
+
+=== Visão Geral dos Requisitos
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+.Conteúdo
+Descrição curta dos requisitos funcionais, forças motrizes, excerto (ou resumo)
+dos requisitos. Link para (esperançosamente existentes) documentos de requisitos
+(com número da versão e informações sobre onde encontrá-lo).
+
+.Motivação
+Do ponto de vista dos usuários finais, um sistema é criado ou modificado para
+melhorar o suporte e/ou melhorar a qualidade de uma atividade negocial.
```

Comment:
Maybe from "atividade negocial" to "atividade de negócios"

Reply:
I'll keep "atividade negocial" as "atividade de negócios" is the corresponding adjective phrase